subprojects/docs/src/docs/userguide/javaPlugin.adoc

```diff
@@ -863,6 +863,7 @@ When using JUnit, we scan for both JUnit 3 and 4 test classes. If any of the fol
 * Class or a super class is annotated with `@RunWith`
 * Class or a super class contain a method annotated with `@Test`
 
+When using http://junit.org/junit5/docs/current/user-guide[JUnit Platform and JUnit Jupiter], we scan for methods annotated with annotations of `org.junit.jupiter.api` package.
```

Comment:
I'd reformulate that sentence a little. I don't necessarily like the use of "we" as it is very vague who is meant by that. Instead maybe say something like this: "Gradle scans for methods annotated with `org.junit.jupiter.api.Test` if the test task has been configured to use JUnit Platform[link]."

Reply:
Somewhere in the Java plugin documentation I'd still like to see some note on what features of JUnit 5 are not supported yet. Just adding it to the release notes won't be enough.